### PR TITLE
add post about the IDR hackathon

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: _posts/2020-02-08-idr-galaxy-hackathon.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+With this integration, Galaxy users would be able to access the images stored at the IDR and automatically
+download them to their Galaxy workspace to run an imaging workflow afterwards.
+
+This integration benefits the life science communities making images easily accessible and reusable
+to every user. It also tightens the collaboration between EOSC-Life Work Package 1 on FAIR data resources,
+Work Package 2 on tools collaboratory and the Demonstrator 6 as part of Work Package 3.
+
+The first outcome of this hackathon is a
+[Galaxy tool to automatically retrive Images from IDR](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/idr_download_by_ids/idr_download_by_ids/0.9).
```
Comment (beatrizserrano) [inline _posts/2020-02-08-idr-galaxy-hackathon.md]:
```suggestion
[Galaxy tool to automatically retrieve images from the IDR](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/idr_download_by_ids/idr_download_by_ids/0.9).
```

**Comment by bgruening** [comment]
@beatrizserrano @wm75 I'm ready? More links needed?

File: _posts/2020-02-08-idr-galaxy-hackathon.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- denbi
+- elixir
+- corbel
+- idr
+- eosc
+- eubi
+---
+
+
+As part of the training program in Work Package 9, CORBEL offers staff visits across research infrastructures.
```
Comment (beatrizserrano) [inline _posts/2020-02-08-idr-galaxy-hackathon.md]:
```suggestion
As part of the training program in Work Package 9, [CORBEL](https://www.corbel-project.eu/home.html) offers staff visits across research infrastructures.
```

File: _posts/2020-02-08-idr-galaxy-hackathon.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+In this frame, EuroBioimaging and ELIXIR members met during three days with the European Galaxy team in Freiburg (Germany)
```
Comment (beatrizserrano) [inline _posts/2020-02-08-idr-galaxy-hackathon.md]:
```suggestion
In this frame, [EuroBioimaging](https://www.eurobioimaging.eu/) and [ELIXIR](https://elixir-europe.org/) members met during three days with the European Galaxy team in Freiburg (Germany)
```

File: _posts/2020-02-08-idr-galaxy-hackathon.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+to make image data from the Image Data Resource (IDR) accessible through the Galaxy interface.
```
Comment (beatrizserrano) [inline _posts/2020-02-08-idr-galaxy-hackathon.md]:
```suggestion
to make image data from the [Image Data Resource](https://idr.openmicroscopy.org/) (IDR) accessible through the Galaxy interface.
```

File: _posts/2020-02-08-idr-galaxy-hackathon.md (hunk 3 above)
Comment (beatrizserrano) [inline _posts/2020-02-08-idr-galaxy-hackathon.md]:
```suggestion
In this frame, EuroBioimaging and ELIXIR members met during three days with the [European Galaxy team](https://galaxyproject.eu/freiburg/) in Freiburg (Germany)
```

File: _posts/2020-02-08-idr-galaxy-hackathon.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+With this integration, Galaxy users would be able to access the images stored at the IDR and automatically
+download them to their Galaxy workspace to run an imaging workflow afterwards.
+
+This integration benefits the life science communities making images easily accessible and reusable
+to every user. It also tightens the collaboration between EOSC-Life Work Package 1 on FAIR data resources,
```
Comment (beatrizserrano) [inline _posts/2020-02-08-idr-galaxy-hackathon.md]:
```suggestion
to every user. It also tightens the collaboration between [EOSC-Life](https://www.eosc-life.eu/) Work Package 1 on FAIR data resources,
```

**Pull request merged**